src/main/webapp/index.html

```diff
@@ -7,20 +7,57 @@
    <script src="script.js"></script>
  </head>
  <body>
-   <div id="MainSection-container">
+   <div id="MainSection">
      <h1>Main Section</h1>
    </div>
  
-   <div id="TwitterBot-container">
-     <h1>Twitter Bot</h1>
-   </div>
+   <div id="TwitterBot">
+     <center><h1>Twitter Bot</h1></center>
+     <header>
+         <!-- It adds a profile picture -->
+    <img src=https://s3.amazonaws.com/caamedia/wp-content/uploads/2017/03/JivanAtmanBest-700x450.jpg alt="Eren Yeager's profile picture" class="profile-picture">
+  <div class="profile-name">
+    <!-- It adds a profile name and handle -->
+    <h3>Eren Yeager</h3>
+    <h4>@ErenYeager</h4>
+  </div>
+   <!-- It adds a follow button, which says 'Thank you for following me.' It does this when it is clicked on. -->
```

Comment:
I think we don't want full twitter integration, so some of these features are not needed.